tests/e2e/utils.js

```diff
@@ -160,10 +165,15 @@ export const isBlockInsertedInWidgetsArea = async ( blockName ) => {
  * @param {'wp_template' | 'wp_template_part'} [params.postType='wp_template'] Type of template.
  */
 export async function goToSiteEditor( params = {} ) {
-	return await visitAdminPage(
-		'site-editor.php',
-		addQueryArgs( '', params )
-	);
+	await visitAdminPage( 'site-editor.php', addQueryArgs( '', params ) );
+
+	if (
+		GUTENBERG_EDITOR_CONTEXT === 'gutenberg' &&
```

Comment:
Should we add a comment as a reminder to remove this check when WordPress 6.2 will be released?

Reply:
Sure thing. Nice catch <3

Reply:
I added the comment, let's see if the flakiness comes this time.